include/seastar/rpc/rpc.hh

```diff
@@ -487,7 +490,7 @@ private:
 private:
     future<> negotiate_protocol(feature_map map);
     void negotiate(feature_map server_features);
-    future<std::tuple<int64_t, std::optional<rcv_buf>>>
+    future<std::tuple<int64_t, std::optional<uint32_t>, std::optional<rcv_buf>>>
```

Comment:
Better to document what the optional means.

Note: microseconds can overflow 32 bits for an rpc call.

Reply:
It's ~1 hour. Are there handlers that last more?